app/models/division.rb

```diff
@@ -3,4 +3,22 @@ class Division < ActiveRecord::Base
   alias_attribute :super_division, :parent
 
   belongs_to :organization
+
+  #JE: I like to keep a reference for the implicit db attributes here in the model class
+  # create_table :divisions do |t|
+  #   t.references :organization, index: true, foreign_key: true
+  #   t.string :name
+  #   t.text :description
+  #   t.integer :parent_id
+  #   t.references :currency, index: true, foreign_key: true
+  #   t.timestamps null: false
+  # end
+
+
+  # For now the id of a special system root node.
+  # Currently convient as an owning divison of migrated orgs and people, but may not be needed in the long run.
+  # Will revisit once full requirements are more clear.
+  ROOT_ID = 99
```

Comment:
sure.  i could use a class method.  Division.root should perhaps return the instance object, and Division.root_id  could be the id